Syntax.md

```diff
@@ -0,0 +1,164 @@
+
+```html
+<p>The <code>Document</code> has an <dfn data-x="concept-document-https-state" data-export=""
+data-dfn-for="Document">HTTPS state</dfn>...</p>
+```
+
+```html
+<dt><dfn data-x="external resource link" data-lt="external resource link" data-export="">Links to
+external resources</dfn></dt>
+```
+
+```html
+<dt><dfn data-export="" data-dfn-type="selector"><code
+data-x="selector-defined">:defined</code></dfn></dt>
+```
+
+## References to other specifications
+
+To produce the familiar bracketed references to other specifications, e.g.
+
+> There are a number of dynamic selectors that can be used with HTML. This section defines when these selectors match HTML elements. [[SELECTORS]](https://html.spec.whatwg.org/#refsSELECTORS) [[CSSUI]](https://html.spec.whatwg.org/#refsCSSUI)
+
+you can use the `<ref>` void element:
+
+```html
+<p>There are a number of dynamic selectors that can be used with HTML. This section defines when
+these selectors match HTML elements. <ref spec=SELECTORS> <ref spec=CSSUI></p>
+```
+
+These match against a bibliography, which is a manually-maintained and sorted `<dl>` at the end of the source file. Its entries look like the following:
+
+```html
+<dt id="refsSELECTORS">[SELECTORS]</dt>
+<dd><cite><a href="https://drafts.csswg.org/selectors/">Selectors</a></cite>, E. Etemad, T. &Ccedil;elik, D. Glazman, I. Hickson, P. Linss, J. Williams. W3C.</dd>
+```
+
+## Specification variants
+
+There are currently up to five variants of the specification produced on each build:
+
+- [Singlepage](https://html.spec.whatwg.org/)
+- [Multipage](https://html.spec.whatwg.org/multipage/)
+- [Developer's edition](https://html.spec.whatwg.org/dev/)
+- Commit snapshots
+- Review drafts
+
+To selectively include or omit content from these variants, you can use the following attributes on any element:
+
+|Edition            |Include |Omit      |
+|-------------------|--------|----------|
+|Singlepage         |w-html  |w-nohtml  |
+|Multipage          |w-split |w-nosplit |
+|Developer's edition|w-dev   |w-nodev   |
+|Commit snapshots   |w-snap  |w-nosnap  |
+|Review drafts      |w-review|w-noreview|
+
+The most important of these for day-to-day work is **w-nodev**, which excludes content (such as details only interesting to web browser implementers) from the developer's edition.
+
+## Macros
+
+Wattsi has a couple of macros used to insert common text fragments:
+
+* `<!-- NON-NORMATIVE SECTION -->` inserts a "This section is non-normative." notice. ([Example](https://html.spec.whatwg.org/multipage/introduction.html#suggested-reading))
+* `<!--INSERT FINGERPRINT-->` inserts a fingerprint image to indicate a feature that may cause fingerprinting. (See the "[Privacy concerns](https://html.spec.whatwg.org/multipage/introduction.html#fingerprint)" section)
```

Comment:
Also note that there are no spaces around this one.

The other macros are `<!--smalltoc-->` and `<!--toc-->`, which generate the short and full tables of contents, respectively. That's probably not of interest to most contributors to HTML, but if the point is to document the Wattsi syntax, they may deserve a mention.